pkg/lib/editor/js/core-config-setup/core-config-setup.js

```diff
@@ -3,6 +3,7 @@ import * as angular from 'angular';
 const forge = require('node-forge')
 const JSZip = require('jszip')
 const yaml = require('js-yaml')
+const uuid = require('uuid')
```

Comment:
Is this library included in `package.json`?

Reply:
added to `package.json` with `npm install`